test/tests/gen/bracket-tests/bracket-test-5/fail.xml

```diff
@@ -0,0 +1,12 @@
+<?oxygen SCHSchema="bracket-test-5.sch"?>
+<!--Context: p[matches(.,'[\(\)\[\]]')]|th[matches(.,'[\(\)\[\]]')]|td[matches(.,'[\(\)\[\]]')]|title[matches(.,'[\(\)\[\]]')]
+Test: report    matches(.,'[\(\[]\p{Zs}+')
+Message:  element contains an opening bracket immediately followed by a space (e.g. '( ' or '[ '). Should the space be removed?  -->
+<root xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
+<article>
+<p>( <bold>Some</bold> text <italic>in</italic> <xref>parentheses</xref>)</p>
+<td>[          <bold>Some</bold> text <italic>in</italic> <xref>parentheses</xref>]</td>
+<title>(   Dummy</title>
+<title>Text)</title>
+</article>
+</root>
```

Comment:
@rkcook312, is this the kind of thing you were talking about earlier (along with [bracket-test-6/fail.xml](https://github.com/elifesciences/eLife-JATS-schematron/pull/663/files#diff-2bd52449ed4b32d0939b67a4b00e38283c5b06e8c746d9138e9621035ffc772c) below)?

Any cases you can think of that are missing?